prog1.py

```diff
@@ -0,0 +1,28 @@
+import psutil
+import subprocess
+import re
+import time
+
+file_num = 3
+file_size = 4096
+#free_space = 
```

Comment:
It's supported that parameters are specified by the user and passed to the program.
I suggest to use `argparse` Python module to handle parameters passed to the program:
https://docs.python.org/2/howto/argparse.html